File: space-game/Presentation.Web/ts/Classes/Motion/Orbit.ts
import { Distance } from "./Distance";
import { Angle } from "../Angle";

export class Orbit {
    Position: string;
    Eccentricity: number;
    PeriodDays: number;
    MeanAnomalyZero: Angle;
    SemiMajorAxis: Distance;
    LongitudeOfAscendingNode: Angle;
    ArgumentOfPeriapsis: Angle;
    Inclination: Angle;
}

File: Data.ConfigDataProvider/Classes/Orbit.cs
namespace Data.ConfigDataProvider.Classes
{
    public class Orbit
    {
        public Distance SemiMajorAxis { get; set; }
        public double Eccentricity { get; set; }
        public Angle Inclination { get; set; }
        public Angle ArgumentOfPeriapsis { get; set; }
        public Angle LongitudeOfAscendingNode { get; set; }
        public Angle MeanAnomalyZero { get; set; }
    }
}